cli/pcluster/config/validators.py

```diff
@@ -1304,3 +1325,6 @@ def ebs_volume_type_size_validator(section_key, section_label, pcluster_config):
             errors.append("The size of {0} volumes must be at least {1} GiB".format(volume_type, min_size))
 
     return errors, warnings
+
+def getBucketNameFromImportPath(importPath):
```

Comment:
since i believe `import_path` is an S3 URL, I'd prefer this be renamed to `get_bucket_name_from_s3_url` to reflect the more generic usage of the function in `s3_bucket_validator`.